### PR TITLE
added slider for games

### PR DESCRIPTION
**Pull request by sp9324**
Created a Netflix-style carousel to display games.

Preview:
https://github.com/user-attachments/assets/f8a1cb13-c429-4cb7-9205-b51876f85a60

### CODE REVIEW

File: src/pages/home/Cards.js [hunk 1]
```diff
@@ -1,16 +1,43 @@
...
+                }
+            }
+        ]
+    };
 
+    return (
         <div className='container'>
-            <div className='row justify-content-center'>
-
+            {/* <div className='row justify-content-center'> */}
```
Comment (ken1000minus7) [inline src/pages/home/Cards.js:39]: Delete this div if not required anymore, rest lgtm

**Review by ken1000minus7 — CHANGES_REQUESTED** [review]

File: src/pages/home/Cards.js [hunk 2]
```diff
@@ -59,8 +86,8 @@ const Cards = () => {
                     label="Search Words"
                     path='/activities/search-any-word'
                     title='Search Words' />
-
-            </div>
+            </Slider>
+            {/* </div> */}
```
Comment (ken1000minus7) [inline src/pages/home/Cards.js:90]: and this closing as well

**Comment by sp9324** [comment]
Changes made; please check.

**Review by ken1000minus7 — APPROVED** [review]
Lgtm

**Pull request merged**